my-app/src/components/CreatedListDetails/store/actions.ts

```diff
@@ -0,0 +1,24 @@
+export const GET_LIST_DETAILS_SUCCESS = 'GET_LIST_DETAILS_SUCCESS';
+export const GET_LIST_DETAILS_ERROR = 'GET_LIST_DETAILS_ERROR';
+export const GET_LIST_DETAILS_REQUESTED = 'GET_LIST_DETAILS_REQUESTED';
+
+export const getListDetailsSuccess = (listDetails: {}) => {
```

Comment:
Please add specific type instead of `{}`

Reply:
done